python/dgl/dataloading/dataloader.py

```diff
@@ -376,6 +428,7 @@ def _prefetch_for_subgraph(subg, dataloader):
         NID,
         dataloader.device,
         dataloader.pin_prefetcher,
+        dataloader.graph._gpu_caches[0],
```

Comment:
0 and 1 is a little confusing, how about use`_gpu_cache[node]` and `_gpu_cache[edge]` as mentioned above?